src/components/countdown/code/lib/countdown.js

```diff
@@ -0,0 +1,56 @@
+var moment = require('moment');
+var Countdown = {};
+module.exports = Countdown;
+
+var pad = function (number) {
+  number = Math.abs(Math.floor(number));
+  if (number > 9) {
+    return number;
+  } else {
+    return '0' + number;
+  }
+};
+
+var roundTowardsZero = function(number) {
+  if (number > 0) {
+    return Math.floor(number);
+  } else {
+    return Math.ceil(number);
+  }
+};
+
+var isSameDay = function(moment1, moment2) {
+  return moment(moment1).isSame(moment2, 'day');
+};
+
+var MOMENT_DISTANCE_UNIT = 'seconds';
```

Comment:
It's not worth putting this in it's own variable since it will be static, it makes the code harder to read.